src/main/java/com/denizenscript/depenizen/bukkit/bridges/JobsBridge.java

```diff
@@ -1,70 +1,64 @@
+
     @Override
     public void init() {
-        ObjectFetcher.registerWithObjectFetcher(JobsJobTag.class);
-        PropertyParser.registerProperty(JobPlayer.class, JobsJobTag.class);
+        ScriptEvent.registerScriptEvent(JobsJobsPaymentScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsExpGainScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsJoinScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsLeaveScriptEvent.class);
+        ScriptEvent.registerScriptEvent(JobsJobsLevelUpScriptEvent.class);
+        ObjectFetcher.registerWithObjectFetcher(JobsJobTag.class, JobsJobTag.tagProcessor);
         PropertyParser.registerProperty(JobsPlayerProperties.class, PlayerTag.class);
-        TagManager.registerTagHandler(new TagRunnable.RootForm() {
-            @Override
-            public void run(ReplaceableTagEvent event) {
-                tagEvent(event);
-            }
-        }, "jobs");
-        DenizenCore.commandRegistry.registerCommand(JobsCommand.class);
-    }
 
-    public void tagEvent(ReplaceableTagEvent event) {
-        Attribute attribute = event.getAttributes();
-        JobsJobTag j = null;
-        if (attribute.hasParam()) {
-            // Documented below.
-            if (JobsJobTag.matches(attribute.getParam())) {
-                j = attribute.paramAsType(JobsJobTag.class);
+        // <--[tag]
+        // @attribute <jobs>
+        // @returns ListTag(JobsJobTag)
+        // @plugin Depenizen, Jobs
+        // @description
+        // Returns the list of all jobs on the server.
+        // -->
+        TagManager.registerTagHandler(ObjectTag.class, "jobs", (attribute) -> {
+            ListTag jobsList = new ListTag();
+            for (Job job : Jobs.getJobs()) {
+                jobsList.addObject(new JobsJobTag(job));
             }
-            else {
-                attribute.echoError("Could not match '" + attribute.getParam() + "' to a valid job!");
-                return;
-            }
-        }
-        else {
-            // <--[tag]
-            // @attribute <jobs[(<name>)]>
-            // @returns ListTag(JobsJobTag)/JobsJobTag
-            // @plugin Depenizen, Jobs
-            // @description
-            // Returns a list of all known jobs, or the job by the given input name if one is given.
-            // -->
-            ListTag jobList = new ListTag();
-            for (Job jb : Jobs.getJobs()) {
-                jobList.addObject(new JobsJobTag(jb));
-            }
-            event.setReplacedObject(jobList.getObjectAttribute(attribute.fulfill(1)));
-        }
-        if (j == null) {
-            attribute.echoError("Invalid or missing job!");
-            return;
-        }
-        event.setReplacedObject(j.getObjectAttribute(attribute.fulfill(1)));
+            return jobsList;
+        });
```

Comment:
This will break any pre-existing scripts using the `jobs` tag? It needs to still support legacy tags